versions/1.1.0-dev.md

```diff
@@ -179,69 +223,45 @@ info:
   title: Targeted Overlay
   version: 1.0.0
 actions:
-  - target: $.paths['/foo'].get
+  - target: '$.paths["/foo"].get'
```

Comment:
The outer quotes are unnecessary, and why change the style of quotes in examples? Both single and double quotes are fine in YAML.

Applies of course to all of the quote changes below.